filament/src/ColorGrading.cpp

```diff
@@ -257,15 +263,30 @@ inline float3 colorDecisionList(float3 v, float3 slope, float3 offset, float3 po
 }
 
 UTILS_ALWAYS_INLINE
-inline constexpr float3 colorAdjustments(float3 v, float contrast, float saturation) {
+inline constexpr float3 contrast(float3 v, float contrast) {
     // Matches contrast as applied in DaVinci Resolve
-    v = MIDDLE_GRAY_ACEScct + contrast * (v - MIDDLE_GRAY_ACEScct);
+    return MIDDLE_GRAY_ACEScct + contrast * (v - MIDDLE_GRAY_ACEScct);
+}
 
-    // S-2016-001 uses Rec.709 luma coefficients for the ASC CDL, including saturation
-    float3 y = dot(v, LUMA_REC709);
+UTILS_ALWAYS_INLINE
+inline constexpr float3 saturation(float3 v, float saturation) {
+    const float3 y = dot(v, LUMA_REC709);
     return y + saturation * (v - y);
 }
 
+UTILS_ALWAYS_INLINE
+inline float3 vibrance(float3 v, float vibrance) {
+    float r = v.r - max(v.g, v.b);
+    float s = (vibrance - 1.0f) / (1.0f + exp(-r * 3.0f)) + 1.0f;
```

Comment:
use `std::exp` or, better, `using namespace std;` in the method.

Reply:
Ok!